docs/release-notes/cloud/cloud-2022-09.md

```diff
@@ -0,0 +1,89 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Cloud September 2022.
+included_jira_versions: ['2022.Q3.7']
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/6.2.59
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/6.3.14
+---
+
+# Cloud September 2022
+
+These release notes are for the Codacy Cloud updates during September 2022.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Product enhancements
+
+-   Codacy now displays the coverage variation metric with a precision of two decimal places on the [Repository Dashboard](../../repositories/repository-dashboard.md) and [Organization Overview](../../organizations/organization-overview.md), and you can [define quality gates](../../repositories-configure/adjusting-quality-settings.md#gates) with a coverage variation threshold using the same precision.
+
+    The increased precision of the metric reflects the code coverage changes better by reducing the false changes due to rounding errors. (IO-54, IO-56)
+
+    ![Coverage threshold with two decimal places on the Quality settings page](../images/io-56.png)
+
+-   While [configuring a coding standard](../../organizations/using-a-coding-standard.md), there's now a checkbox on the header of the code pattern list that allows toggling all code patterns that are currently visible on the list. This allows you to conveniently toggle code patterns in bulk, for example, to enable all security code patterns. (CY-6336)
```

Comment:
Suggestion to focus on the new capability instead of the new checkbox:

```suggestion
-   While [configuring a coding standard](../../organizations/using-a-coding-standard.md), you can now toggle all code patterns that are currently visible on the list using the new checkbox on the header. This allows you to conveniently toggle code patterns in bulk, for example, to enable all security code patterns. (CY-6336)
```

Reply:
Good idea, thanks @claudiacarpinteiro!